api/proto/config/devconfig.proto

```diff
@@ -90,6 +90,30 @@ message EdgeDevConfig {
 
   // Baseos Config Block
   BaseOS baseos = 26;
+
+  // global_profile, if set, controls set of applications which will run.
+  // The Activate=true app instances which have this profile in their profile_list
+  // will run. If the global_profile is not set, then the profile_list is not
+  // used to gate the application instances.
+  string default_profile = 27;
```

Comment:
Wait, you have it global_profile here in the comment but still default_profile in the actual name of the field, what gives @eriknordmark ?

Reply:
Oops. Fixed.